packages/pulp-rpm/pulp-rpm.spec

```diff
@@ -269,7 +278,25 @@ A collection of yum plugins supplementing Pulp consumer operations.
 %{_usr}/lib/yum-plugins/pulp-profile-update.py*
 %doc LICENSE COPYRIGHT
 
+# ---- Pulp Manifest -------------------------------------------------------------
+%if %{pulp_server}
+%package -n python-pulp-manifest
+Summary: Tool to create a PULP_MANIFEST in a given directory
+Group: Development/Languages
+Provides: python2-pulp-manifest
+Obsoletes: python2-pulp-manifest < %{version}
```

Comment:
What does this line ensure for a new package?

Reply:
This is still an artifact of upstream spec files not being fedora compliant in terms of python naming.  Since downstream packages are compliant, this helps ensure proper upgrade pathways from packages in the fedora repository to the upstream repository